src/service/Afterschool.java

```diff
@@ -0,0 +1,52 @@
+package service;
+import entity.*;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class Afterschool {          // clasa serviciu care expune operatiile sistemului
+
+    // poate nu e nevoie de aceste date membre, de constructor si de getteri si setteri
+    private Student[] students;
+    private List<Employee> employees; // list = to access the elements frequently by using the index
+
+    public Afterschool(Student[] students, List<Employee> employees) {
+        this.students = students;
+        this.employees = employees;
+    }
+
+    public Student[] getStudent() {
+        return students;
+    }
+    public List<Employee> getEmployees(){
+        return employees;
+    }
+    public void setStudent(Student[] students) {
+        this.students = students;
+    }
+    public void setTeacher(List<Employee> employees) {
+        this.employees = employees;
+    }
+
+
+
+    // actiuni / interogari pe obiectele create
+    public void AddTeacher (Employee employee){
+        employees.add(employee);
+    }
+    public void AddStudent (Student student) {}
```

Comment:
toate metodele respecta structura camelCase in general, rugamintea ar fi sa redenumesti aceste metode conform practicii si sa oferi o logica acestor metode pe care sa le apelezi apoi in Main